bootstrap.js

```diff
@@ -0,0 +1,46 @@
+/* global Components */
+
+const { classes: Cc, interfaces: Ci } = Components;
+const DefaultUA = Cc["@mozilla.org/network/protocol;1?name=http"]
+  .getService(Ci.nsIHttpProtocolHandler).userAgent;
+const ObserverService = Cc["@mozilla.org/observer-service;1"].getService(Ci.nsIObserverService);
+
+class UAOverrider {
+  init() {
+    ObserverService.addObserver(this, "http-on-useragent-request", false);
```

Comment:
I don't think we're able to use preprocessor constants in our JS sources, but feel free to enlighten me.

However, you raise a good point. I've moved the string to a const so we only have to change a single place if the topic ever changes.